game/src/Credits/Credits.gd

```diff
@@ -0,0 +1,98 @@
+extends Control
+
+signal backButtonPressed
+
+###############
+# Credits CSV format
+# The project title row is the only requirement within the csv file, however
+# it can be on any row, so long as it exists.
+# ----------------------
+# title,project-title
+# role-name,person-name
+# role-name,person-name
+# role-name,person-name
+# ...
+###############
+
+@export var coreCreditsPath : String = 'res://common/credits.csv'
```

Comment:
```suggestion
@export_file("*.csv")
var core_credits_path : String = 'res://common/credits.csv'
```

This should probably be a custom CSVLoader resource instead to be honest, but I'll leave that for later implementation.